include/point.h

```diff
@@ -11,8 +11,12 @@ class Point {
   //! Constructor with id and coordinates
   //! \param[in] id index of the vertex
   //! \param[in] coord Coordinates of the point
-  Point(unsigned id, const std::array<double, Tdim>& coord) : id_{id} {
```

Comment:
Revert to `Point(unsigned id, const std::array<double, Tdim>& coord) : id_{id} {`